scripts/deploy-html/index.js

```diff
@@ -22,8 +22,8 @@ Promise.resolve(PUBLIC_DIR)
       .catch(err => console.log('Error in Promise', err))
   })
 
-function readFile (filepath) {
-  return new Promise(function (resolve, reject) {
+function readFile(filepath) {
+  return new Promise(function(resolve, reject) {
```

Comment:
__[eslint]__
* Error - 'Promise' is not defined. (no-undef)

<!-- Comment by [SideCI](https://sideci.com) -->